rpm/SOURCES/openresty.init

```diff
@@ -73,7 +73,7 @@ configtest() {
 }
 
 rh_status() {
-    status $nginx
+    status -p "$pidfile" $nginx
```

Comment:
I'm afraid the `-p` option is *not* really portable. We are risking breaking portability here. If the user wants to run multiple OpenResty services, then she shouldn't use the default init script as-is anyway.